nuxeo-liveconnect-dropbox-core/src/main/java/org/nuxeo/ecm/liveconnect/dropbox/DropboxBlobProvider.java

```diff
@@ -75,18 +79,17 @@ public URI getURI(ManagedBlob blob, UsageHint usage, HttpServletRequest servletR
         LiveConnectFileInfo fileInfo = toFileInfo(blob);
         String filePath = fileInfo.getFileId();
         String url = null;
-        DbxClient client = getDropboxClient(getCredential(fileInfo));
+        DbxClientV2 client = getDropboxClient(getCredential(fileInfo));
         try {
             switch (usage) {
             case STREAM:
-                url = client.createTemporaryDirectUrl(filePath).url;
+                url = client.files().getTemporaryLink(filePath).getLink();
                 break;
             case DOWNLOAD:
-                url = client.createShareableUrl(filePath);
-                url = url.replace("dl=0", "dl=1"); // enable download flag in url
+                url = client.files().getTemporaryLink(filePath).getLink();
```

Comment:
I tried to use the new `createSharedLinkWithSettings()` function (similiar to the former `createSharableUrl`but the  the inline preview of PDFs doesnt work.
with `getTemporaryLink` works nicely.